README.md

```diff
@@ -59,7 +59,7 @@ We've put together a seperate `README` file that should help to get you up and r
 ## Conventions
 
 ### Github Conventions
-- We follow the Forking Workflow: here is what it is [here](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) and how to use it [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
+- We follow the Forking Workflow: [here](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow) and how to use it [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) is what it is, and [here](https://gist.github.com/Chaser324/ce0505fbed06b947d962) is a good TLDR of it. 
```

Comment:
It took me a while to understand the sentence and which link belongs to which; all the links are in the same sentence and there's just a lot of "here" in one sentence. 

I would suggest breaking it up into bullet points, e.g.

- We follow the Forking Workflow:
    - [what it is](https://www.atlassian.com/git/tutorials/comparing-workflows#forking-workflow)
    - [how to use it](https://gist.github.com/Chaser324/ce0505fbed06b947d962)
    - [TLDR](https://gist.github.com/Chaser324/ce0505fbed06b947d962)

Also, the link for "TLDR" and "how to use it" are the same.

Reply:
Wow, I must still have been sick when I opened this :laughing:. Great points, I've moved to your suggested format and removed the last "TLDR" link.

@mingyokim - could you re-review?